webapp/locales/ru_RU.json

```diff
@@ -0,0 +1,2246 @@
+{
```

Comment:
@Mogge this will have no effect. The file name has the extension `_RU` to it and our software won't recognize it.

Reply:
OK. I try to change the name of the language in lokalise.

Reply:
@Mogge can you remove this file from the commit? So we can merge both this PR and https://github.com/Human-Connection/Human-Connection/pull/2277?

Reply:
I'll approve then

Reply:
Done